flows/nodes/template.md

```diff
@@ -79,3 +84,22 @@ Running: foobar ("foo to the bar")
 Scheduled every 1h0m0s
 Last Run: 2023-02-16 20:01:45.606580068 +0000 UTC
 ```
+
+## Example - Extracting Query Values
+
+The Text template can index into result data and extract specific values, this can be useful when using query result values for high level fields.  This example shows running a query that looks for users with excessive login failures and sends an email with the user and failure rates in the subject line.
```

Comment:
```suggestion
The Text template can index into result data and extract specific values. This can be useful when using query result values for high level fields. This example shows running a query that looks for users with excessive login failures and sends an email with the user and failure rates in the subject line.
```